test/unit/specs/mount/options/slots.spec.js

```diff
@@ -4,6 +4,18 @@ import Component from '~resources/components/component.vue'
 import ComponentWithSlots from '~resources/components/component-with-slots.vue'
 
 describe('mount.slots', () => {
+  let _window
+
+  beforeEach(() => {
+    _window = window
+  })
+
+  afterEach(() => {
+    /* eslint-disable */
```

Comment:
Unless I'm mistakenm you can just add // eslint-disable-line to the line, instead of 2 comments. Even better, you can add `//eslint-disable-line name-of-rule` to only disable eslint for the rule which is failing

Reply:
Thank you for pointing out.
I fixed it.